packages/web-components/src/components/va-search/va-search.css

```diff
@@ -0,0 +1,53 @@
+@import '../../mixins/buttons.css';
+@import '../../mixins/inputs.css';
+
+:host {
+  display: block;
+}
+
+#va-search-form {
+  align-items: center;
+  display: flex;
+}
+
+#va-search-input {
+  border-color: var(--color-gray);
+  display: inline-block;
+  height: 4.8rem;
+  margin: 0;
+  max-width: none;
+  min-width: 24.8rem;
+  padding: 1.2rem 0.8rem;
+}
+
+#va-search-input:focus {
+  outline: 2px solid var(--color-gold-light);
+  outline-offset: 2px;
+  z-index: 1;
+}
```

Comment:
Could we add the `input` selector to `focusable.css` instead?

https://github.com/department-of-veterans-affairs/component-library/blob/3c44d4a5e3b134c8d8be692fde19caa9e708cdda/packages/web-components/src/mixins/focusable.css#L3-L9